1.10/networking/virtual-networks/index.md

```diff
@@ -5,73 +5,60 @@ feature_maturity: preview
 menu_order: 04
 ---
 
-DC/OS supports Container Networking Interface (CNI)-compatible virtual networking solutions, including [Calico](https://github.com/dcos/examples/tree/master/calico) and Contrail. DC/OS also provides a native virtual networking solution called DC/OS Overlay. 
+DC/OS supports Container Networking Interface (CNI)-compatible virtual networking solutions, including [Calico](https://github.com/dcos/examples/tree/master/calico) and Contrail. DC/OS also provides a native virtual networking solution called DC/OS Overlay.
 
-# DC/OS Overlay
-DC/OS Overlay provides an IP per container for Mesos and Docker containers alike. DC/OS Overlay uses CNI (Container Network Interface) for the [Mesos Containerizer](http://mesos.apache.org/documentation/latest/containerizers/#Mesos) and Docker libnetwork for the [Docker Containerizer](http://mesos.apache.org/documentation/latest/containerizers/#Docker).
+# Virtual Networks on DC/OS
```

Comment:
We probably need to re-write this entire section. This section is very specific to DC/OS overlay. We need an intro section over which talks about container networking in DC/OS and how DC/OS enables it using CNI and CNM standards. We can then have sections on CNI and CNM and DC/OS overlay.

Reply:
I will try to share some content on a word doc for this section later tonight.